site/tests/universal/page-header.test.ts

```diff
@@ -1,6 +1,6 @@
 import { expect, test } from "@playwright/test";
 
-test("page header navigation works", async ({ page }) => {
+test("links and interactions in page header should work", async ({ page }) => {
```

Comment:
How about placing search modal into another `test("")` (same file)? It might be easier to communicate the intent of our app interactions.

Reply:
Sounds good. Updated here => https://github.com/blockprotocol/blockprotocol/pull/443/commits/54e97e8463a588161ddf075be0d38afd4331ced6